test/recursive-raddir-test.js

```diff
@@ -72,4 +72,20 @@ describe('readdir', function() {
       done()
     })
   })
+
+  it('works when negated ignore list is given', function(done) {
+
+    var expectedFiles = [
+      __dirname + '/testdir/c.txt',
```

Comment:
Can you use `path.join` instead of string concatenation so this works on windows as well?